jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -320,7 +334,7 @@ public void reconnectingIn(int i)
                 }
             }
 
-            @Override
+            @Deprecated
```

Comment:
This method has been removed from smack. Since it contains some functionality I've only marked it as deprecated.
Was that correct? Or should we keep the method regardless of what smack did?